app/helpers/application_helper.rb

```diff
@@ -1,2 +1,3 @@
 module ApplicationHelper
+	
```

Comment:
We don't need the newline, lets keep the git history clean. :) please remove this line.